src/components/structures/MatrixChat.tsx

```diff
@@ -1993,14 +2019,15 @@ export default createReactClass({
                  */
                 const LoggedInView = sdk.getComponent('structures.LoggedInView');
                 view = (
-                    <LoggedInView ref={this._collectLoggedInView} matrixClient={MatrixClientPeg.get()}
-                        onRoomCreated={this.onRoomCreated}
-                        onCloseAllSettings={this.onCloseAllSettings}
-                        onRegistered={this.onRegistered}
-                        currentRoomId={this.state.currentRoomId}
-                        showCookieBar={this.state.showCookieBar}
-                        {...this.props}
-                        {...this.state}
+                    <LoggedInView ref={this.loggedInView}
+                                  matrixClient={MatrixClientPeg.get()}
+                                  onRoomCreated={this.onRoomCreated}
+                                  onCloseAllSettings={this.onCloseAllSettings}
+                                  onRegistered={this.onRegistered}
+                                  currentRoomId={this.state.currentRoomId}
+                                  showCookieBar={this.state.showCookieBar}
+                                  {...this.props}
+                                  {...this.state}
```

Comment:
indentation says we don't align these. 

WebStorm does the right thing if the `ref` is shoved onto its own line (no attrs on the tag line)

Reply:
I tried to leave existing styling as much as possible, but will change this